artiq/coredevice/comm_moninj.py

```diff
@@ -84,9 +84,8 @@ async def _receive_cr(self):
                 if not ty:
                     return
                 if ty == b"\x00":
-                    payload = await self._reader.readexactly(9)
-                    channel, probe, value = struct.unpack(
-                        self.endian + "lbl", payload)
+                    payload = await self._reader.readexactly(13)
+                    channel, probe, value = struct.unpack(self.endian + "lbq", payload)
```

Comment:
How does this belong in this PR?

Reply:
I thought we had agreed a long time ago on how to structure this.

Reply:
hmm, since the byte sizes are expanded in firmware, without this line artiq_rtiomon and subsequently artiq_dashboard will not work. Should we further cherrypick this away?

Reply:
It's OK for a PR to depend on another one. And anyway I was expecting the probe width expansion one to be straightforward and easily merged but obviously that was wrong...